src/Client.php

```diff
@@ -401,6 +402,33 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             return null;
         }
 
+        $responseParsers = [
+            'application/json' => [$this, 'parseJson'],
+            'application/pdf' => [$this, 'parsePdf'],
+        ];
+
+        if (preg_match('/^(\w+\/\w+).*/i', $response->getHeaderLine('Content-Type'), $matches)) {
```

Comment:
`\w` - A "word" character is any letter or digit or the underscore character, that is, any character which can be part of a Perl "word". The definition of letters and digits is controlled by PCRE's character tables, and may vary if locale-specific matching is taking place. For example, in the "fr" (French) locale, some character codes greater than 128 are used for accented letters, and these are matched by \w.

as it allows even numbers maybe we can use `[a-zA-Z]` ?